packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { Button } from "react-aria-components";
+
+import { WalkthroughInfo } from "../../../data/src/useWalkthroughData";
+
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps {
+  id: string;
+  data: WalkthroughInfo;
+  href?: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  href,
+}: WalkthroughCardProps) {
+  const router = useRouter();
+
+  const handleClick = () => {
+    if (href) router.push(href);
+  };
+
+  const canDisplayCard =
+    data && data.title && data.subtitle && data.description;
+
+  if (!canDisplayCard) return null;
+
+  return (
+    <Button
+      className="ui-WalkthroughCard--CardContainer"
+      onPress={handleClick}
+      aria-label={`${id}-walkthrough-card`}
+    >
+      <header className="ui-WalkthroughCard--Title">{data.title}</header>
```

Comment:
Also I'm not a semantic expert, but I **_think_** the structure we want here is something like:

```html
<article>
  <header>
    <h2/h3> title -- depends on where it's being used
    <p> subtitle
  </header>
  <p> description
</article>
```